front/app/components/UI/PopContainer/index.tsx

```diff
@@ -66,28 +66,23 @@ const StyledIcon = styled(Icon)`
   fill: #333;
 `;
 
```

Comment:
Was in this file while debugging, so quickly refactored. Nothing related to the bug.